model/src/main/java/org/cloudfoundry/identity/uaa/provider/OIDCIdentityProviderDefinition.java

```diff
@@ -90,6 +104,7 @@ public boolean equals(Object o) {
         if (this.passwordGrantEnabled != that.passwordGrantEnabled) return false;
         if (this.setForwardHeader != that.setForwardHeader) return false;
         if (this.jwtClientAuthentication != that.jwtClientAuthentication) return false;
+        if (this.additionalAuthzParameters != that.additionalAuthzParameters) return false;
```

Comment:
Please use `equals` instead (also for the `jwtClientAuthentication` one line above)